app/controllers/static_pages_controller.rb

```diff
@@ -8,9 +8,6 @@ def contact_us
   def cookies
   end
 
-  def maths_and_physics
```

Comment:
The actual view file can also be removed.

Reply:
Arrrrgh. Again, I thought I'd done this!

Reply:
Yeah, was another bad rebase, because we'd edited the view file recently

Reply:
One habit I've got into is to always eyeball my PRs on GitHub after any changes/rebases just to make sure I've not missed anything obvious. It saves this sort of back and forth with a reviewer for stuff that I could/should have spotted.